docs/cloud/features/alerts_notifications.md

```diff
@@ -27,34 +49,68 @@ Tobiko Cloud Alerts can be triggered by the following events:
 
-# Notification Targets
+![Image showing the add measure Alert page condition section](./alerts_notifications/add_measure_alert_condition.png)
 
-Notification targets allow you to configure different ways for alerts to contact you. They can be used by multiple alerts so you only have to configure them once.
+Now specify the alert Artifact field.
 
-## Supported Notification Target Types
+Some measures, like run time, are most useful when accumulated over an entire `plan` or `run`. For example, you might want to alert whenever a `run`'s total run time is longer than four hours.
 
-Tobiko Cloud supports the following notification target types:
+Configure a cumulative measure alert by choosing an Artifact type of Plan or Run.
 
-- Slack API
-- Slack Webhook
-- PagerDuty
+Other measure alerts are useful when applied to individual models or steps in a `plan` or `run`. For example, you might know that your `users` model should process around 1000 new users each day. You could create a measure alert that notifies you if the number of users drops below 500.
+
+Configure a non-cumulative measure alert by choosing an Artifact type of Measure.
+
+![Image showing the add measure Alert page Artifact field](./alerts_notifications/add_measure_alert_artifact.png)
+
+To prevent alert fatigue, you can limit measure-based alerts to a specific environment  or model in the optional Environment and Model fields.
+
+![Image showing the add measure Alert page](./alerts_notifications/add_measure_alert_page.png)
+
+## Notification Targets
 
-## Notification Target Configuration
+Each alert is sent to one or more notification targets.
 
-Notification targets can be configured in the Tobiko Cloud Settings section.
+A notification target is a way for alerts to contact you. A target can be used in multiple alerts, so you only have to configure them once.
 
-To add a new notification target, click on Add Notification Target in the top right.
+### Notification Target Configuration
+
+Configure Notification targets in the Tobiko Cloud Settings section.
+
+To add a new notification target, navigate to the Notification Targets page and click the Add Notification Target button in the top right.
 
 ![Image highlighting location of the Add Notification Target button](./alerts_notifications/add_notification_target_button.png)
 
-Then enter a descriptive name for the new notification target, select its type, fill in the configuration information, and click Save.
+Then enter a descriptive name for the new notification target, select its type (described [below](#notification-target-types)), fill in the configuration information, and click Save.
 
 ![Image showing the add Notification Target page](./alerts_notifications/add_notification_target_page.png)
+
+### Notification target types
+
+Tobiko Cloud supports the following notification target types, which require you to provide different pieces of configuration information.
+
+- Slack API
```

Comment:
Add examples of what these inputs should look and feel like. Can be in text form.